Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -453,6 +453,7 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
         let buttonPosition = sender.convert(CGPoint(), to: tableView)
         if let indexPath = tableView.indexPathForRow(at: buttonPosition), let newQuery = suggestions?[indexPath.row] {
             searchDelegate?.searchViewController(self, didAppend: newQuery + " ")
+            searchQuery = newQuery + " "
```

Comment:
Something to consider:

For 1st query if you keep appending and it will add a space to query and you'll have multiple spaces


https://user-images.githubusercontent.com/8919439/112387755-87863700-8cc8-11eb-82ca-574918acc654.mov

Reply:
Oh good point, that append shouldn't even be on the first one